legal-api/src/legal_api/models/db.py

```diff
@@ -15,13 +15,279 @@
 
 These will get initialized by the application using the models
 """
-from flask_sqlalchemy import SQLAlchemy
+from datetime import datetime
+
+from flask import current_app
+from flask_sqlalchemy import SignallingSession, SQLAlchemy
+from sql_versioning import TransactionManager, debug
+from sql_versioning import disable_versioning as _new_disable_versioning
+from sql_versioning import enable_versioning as _new_enable_versioning
+from sql_versioning import version_class as _new_version_class
+from sqlalchemy import event, orm
+from sqlalchemy.orm import Session, mapper
 from sqlalchemy_continuum import make_versioned
+from sqlalchemy_continuum import version_class as _old_version_class
+from sqlalchemy_continuum.manager import VersioningManager
 
 
 # by convention in the Flask community these are lower case,
 # whereas pylint wants them upper case
 db = SQLAlchemy()  # pylint: disable=invalid-name
 
-# make_versioned(user_cls=None, plugins=[FlaskPlugin()])
-make_versioned(user_cls=None)
+
+class Transaction(db.Model):
```

Comment:
Introduce custom `Transaction` model to solve FK related issues between `filings` and `transaction` tables when we completely remove old versioning and only new versioning exists. To make sure there's no conflicts when two versionings exist in the code, I make them use this custom model though they both have their own. And the removal of old versioning will be clean and the core logic of new versioning won't be touched then.